### ISSUE TITLE
feat: Add mermaid(Charts) for Databend Docs

### ISSUE THREAD

**Issue by Chasen-Zhang** [datafuselabs/databend-docs#985]
## Easier to draw charts in markdown
refs: https://mermaid.js.org/syntax/flowchart.html
https://docusaurus.io/docs/markdown-features/diagrams

## Demo
```
graph TD;
    A-->B;
    A-->C;
    B-->D;
    C-->D;
 ```
![image](https://github.com/user-attachments/assets/a2ee4f62-8138-4f23-b944-39bb9cdbd874)

```
sequenceDiagram
    participant Alice
    participant Bob
    Alice->>John: Hello John, how are you?
    loop HealthCheck
        John->>John: Fight against hypochondria
    end
    Note right of John: Rational thoughts <br/>prevail!
    John-->>Alice: Great!
    John->>Bob: How about you?
    Bob-->>John: Jolly good!

```
![image](https://github.com/user-attachments/assets/3f546e8c-8664-4ea6-bf65-0448a8a7c3b7)
  
 ```
pie title Pets adopted by volunteers
    "Dogs" : 386
    "Cats" : 85
    "Rats" : 15
```

![image](https://github.com/user-attachments/assets/b51f1ba5-3451-4d90-9de2-903d64704835)

**Comment by Chasen-Zhang** [comment]
fix: #986

**Comment by Chasen-Zhang** [comment]
@soyeric128  可以用这个尝试下
